taxcalc/taxcalcio.py

```diff
@@ -47,11 +47,11 @@ class TaxCalcIO(object):
         or string is name of optional ASSUMP file.
 
     growdiff_response: Growdiff object or None
-        growdiff_response Growdiff object used in dynamic analysis;
-        must be None when conducting static analysis.
+        growdiff_response Growdiff object used only in dynamic analysis;
+        must be None when not conducting dynamic analysis.
```

Comment:
@MattHJensen said:
> Not sure whether it is worth splitting hairs here in the code, but growdiff_response must also be None when conducting a partial-equilibrium dynamic analysis.

I thought your taxonomy was: static, partial-equilibrium(ie, behavior), and dynamic analysis, right?  I thought this was the implication of the various quotes you offered last week.  If I'm still confused, we need to talk on the phone.

Given that analysis taxonomy, non-dynamic analysis is static or partial-equilibrium, and the wording is sensible.